plugins/SitesManager/API.php

```diff
@@ -1498,10 +1498,7 @@ public function updateSite(
             $settingValues = [];
         }
 
-        if (empty($coreProperties)) {
```

Comment:
The variable is not defined anywhere before, so there is no value in only defining it conditionally.